lib/mix/tasks/nimble.phx.gen.template.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Mix.Tasks.Nimble.Phx.Gen.Template do
+  @shortdoc "Generates Nimble's Phoenix template"
+
+  use Mix.Task
+
+  alias Nimble.Phx.Gen.Template.{Project, Template}
+
+  @version Mix.Project.config()[:version]
+  @switches [api: :boolean, web: :boolean]
+
+  def run([version]) when version in ~w(-v --version) do
+    Mix.shell().info("Nimble.Phx.Gen.Template v#{@version}")
+  end
+
+  def run(args) do
+    if Mix.Project.umbrella?() do
+      Mix.raise("mix nimble.phx.gen.template can only be run inside an application directory")
+    end
+
+    {opts, _params} = parse_opts(args)
+
+    %Project{
+      is_api_project: opts[:api],
+      otp_app: Mix.Phoenix.otp_app(),
+      base_module: Mix.Phoenix.base(),
+      web_module: Mix.Phoenix.inflect("_DUMMY_")[:web_module],
```

Comment:
Please explain what this is `_DUMMY_` does?

Reply:
Added the comment [here](https://github.com/nimblehq/elixir-templates/pull/22/files#diff-440d7b360941f68ab39541b0a23aa4ffR22) @olivierobert

Reply:
Yes, correct @bterone :D will change that to `Module.concat(["#{Mix.Phoenix.base()}Web"])` 😃